pages/general-information-and-resources/glossary.md

```diff
@@ -100,6 +100,8 @@ cSpell: ignore Technica
 
 - **CO** - Contracting Officer. A part of acquisitions.
 
+- **COTS** - Commercial Off The Shelf software, as opposed to custom-built software
```

Comment:
COTS is a general term, not just for software. What do you think about generalizing this?

```suggestion
- **COTS** - Commercial Off The Shelf, as opposed to a custom-built product
```